ci-operator/config/openshift/aws-load-balancer-operator/openshift-aws-load-balancer-operator-main.yaml

```diff
@@ -93,6 +93,26 @@ tests:
         requests:
           cpu: 100m
     workflow: aws-load-balancer-install-rosa
+- as: e2e-aws-proxy-operator
+  optional: true
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index-aws-load-balancer-operator-bundle
+    env:
+      OO_CHANNEL: stable-v0
+      OO_INSTALL_NAMESPACE: aws-load-balancer-operator
+      OO_PACKAGE: aws-load-balancer-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: test
+      cli: latest
+      commands: make test-e2e
```

Comment:
Aren't there supposed to be some exported env variables for this test?

Reply:
No, OLM will setup the environment variables on all the operators automatically, if the cluster wide egress proxy is set up on the cluster (which is the case).